digital-twin/e2e-tests/digital-twin-e2e-tests-common/src/main/java/com/microsoft/azure/sdk/iot/digitaltwin/e2e/tests/DigitalTwinCommandE2ETests.java

```diff
@@ -54,17 +67,18 @@
 @RunWith(Parameterized.class)
 public class DigitalTwinCommandE2ETests {
     private static final String IOT_HUB_CONNECTION_STRING = retrieveEnvironmentVariableValue(IOT_HUB_CONNECTION_STRING_ENV_VAR_NAME);
-    private static final String TEST_INTERFACE_INSTANCE_NAME_1 = retrieveInterfaceNameFromInterfaceId(TestInterfaceInstance1.TEST_INTERFACE_ID);
-    private static final String TEST_INTERFACE_INSTANCE_NAME_2 = retrieveInterfaceNameFromInterfaceId(TEST_INTERFACE_ID);
+    private static final String TEST_COMPONENT_NAME_1 = retrieveComponentNameFromInterfaceId(TestComponent1.TEST_INTERFACE_ID);
```

Comment:
Should be `retrieveComponentNameFromComponentId`, but I can refactor these when I make the updates for the service client.